packages/core/src/App/Containers/RealAccountSignup/add-or-manage-accounts.jsx

```diff
@@ -12,185 +12,179 @@ import LoadingModal from './real-account-signup-loader.jsx';
 import 'Sass/add-or-manage.scss';
 import 'Sass/change-account.scss';
 
-class AddOrManageAccounts extends React.Component {
-    static contextType = PlatformContext;
-    constructor(props) {
-        super(props);
-        this.state = {
-            active_index: this.props.has_fiat && this.props.available_crypto_currencies?.length === 0 ? 1 : 0,
-            finished: undefined,
-            form_error: '',
-            form_value: {
-                crypto: '',
-                fiat: '',
-            },
-        };
-    }
+const AddOrManageAccounts = props => {
+    const { is_deriv_crypto } = React.useContext(PlatformContext);
 
-    async componentDidMount() {
-        this.props.setLoading(true);
-        await WS.mt5LoginList();
-        this.props.setLoading(false);
-    }
+    const {
+        available_crypto_currencies,
+        can_change_fiat_currency,
+        createCryptoAccount,
+        currency,
+        current_currency_type,
+        has_fiat,
+        is_loading,
+        onError,
+        onSuccessSetAccountCurrency,
+        setCurrency,
+        setLoading,
+    } = props;
```

Comment:
Let's move this to the component arguments.

Reply:
The reason i did not destructure the props in component argument is that we need to pass all the props to `ChangeAccountCurrency ` and 'AddCryptoCurrency', in that case we need to pass them one by one

Reply:
Understood. nice.